core/src/components/table/table-body-row/table-body-row.tsx

```diff
@@ -34,27 +45,24 @@ export class TdsTableBodyRow {
 
   tableEl: HTMLTdsTableElement;
 
-  bodyCheckBoxClicked(event) {
-    const row = this.host;
-    this.bodyCheckBoxStatus = event.currentTarget.checked;
-    if (this.bodyCheckBoxStatus === true) {
-      row.classList.add('tds-table__row--selected');
-    } else {
-      row.classList.remove('tds-table__row--selected');
-    }
-    this.internalTdsRowChange.emit(this.bodyCheckBoxStatus);
-  }
-
-  bodyCheckBoxStatusUpdater(status) {
-    this.mainCheckBoxStatus = status;
-    this.bodyCheckBoxStatus = this.mainCheckBoxStatus;
-    const row = this.host;
-    if (this.bodyCheckBoxStatus === true) {
-      row.classList.add('tds-table__row--selected');
-    } else {
-      row.classList.remove('tds-table__row--selected');
-    }
-    this.internalTdsRowChange.emit(this.bodyCheckBoxStatus);
+  /**  Send status of single row to the parent, tds-table component that hold logic for data export and main checkbox control */
+  @Event({
+    eventName: 'tdsSelect',
+    composed: true,
+    cancelable: false,
+    bubbles: true,
+  })
+  tdsSelect: EventEmitter<{
+    tableId: string;
```

Comment:
I don't think the tableId is necessary since we can listen on the table or row itself, but I'm fine with keeping it.